crates/bevy_core/src/name.rs

```diff
@@ -17,7 +17,7 @@ use std::{
 /// [`Name`] should not be treated as a globally unique identifier for entities,
 /// as multiple entities can have the same name.  [`bevy_ecs::entity::Entity`] should be
 /// used instead as the default unique identifier.
-#[derive(Reflect, FromReflect, Component, Debug, Clone)]
+#[derive(Reflect, FromReflect, Component, Clone)]
```

Comment:
Nit: While we're here we could also register `#[reflect(Debug)]` so that this value is used when debugging `Name` as a `dyn Reflect`.